src/lib/ledger_catchup/ledger_catchup.ml

```diff
@@ -4,6 +4,31 @@ open Protocols.Coda_transition_frontier
 open Pipe_lib
 open Coda_base
 
+(** [Ledger_catchup] is a process that connects a foreign external transition
```

Comment:
it's not a separate OS process, so maybe "procedure" or "algorithm" is a better word